cookbooks/fb_users/README.md

```diff
@@ -200,21 +206,23 @@ in `GID_MAP`.
 For all other values we accept, they will only be passed to the resource if
 they exist in the user's hash.
 
-### Helper methods
+### Removing users or groups
+To remove a user or group, set the `action` to `:delete` and the user or group
+will be automatically cleaned up during the chef run. To automatically clean up
+a user's home directory while removing the user from the system, leave
+`manage_home` set to `true`.
 
+### Helper methods
 This cookbook provides a few helper methods for your convenience:
 
 #### FB::Users.initialize_group(node, groupname)
-
 Since initializing a group nicely involves setting `members` to an empty array
```

Comment:
Please don't delete these empty lines. First, MDL says there should be a blank line. Second, having it allows one to `gqap` in vim to autoformat/wrap. Otherwise it gets wrapped *with* the title/heading.